firefox-ios/nimbus-features/#menuRefactorFeature.yaml#

```diff
@@ -0,0 +1,18 @@
+# The configuration for the menuRefactorFeature feature
+features:
+  menu-refactor-feature:
+    description: >
+      Controls the menu refactor feature
+    variables:
+      enabled:
+        description: >
+          Controls which menu users will see
+        type: Boolean
+        default: false
+    defaults:
+      - channel: beta
+        value:
+          new-variable: false
```

Comment:
I don't see `new-variable` defined in variables - should this be `enabled` instead?

Reply:
Also the name of this file looks off.